mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping

```diff
@@ -0,0 +1,3 @@
+CLASS wv net/minecraft/resource/ObjectPassingResourceReloadListener
```

Comment:
This name is quite verbose. I believe the term `supplier` fits pretty well in this context; could it be used in the name somehow?

Reply:
I'm not sure. SupplyingResourceReloadListener?